_includes/misc/mobile_platform_compatibility.md

```diff
@@ -11,4 +11,6 @@
 | iOS 12    | Safari  | iPhone/iPad  |
 | iOS 13    | Safari  | iPhone/iPad  |
 
-Please report platform issues and bugs in the [{{site.productname}} issue tracker](https://github.com/tinymce/tinymce/issues).
\ No newline at end of file
+Please report platform issues and bugs in the [{{site.productname}} issue tracker](https://github.com/tinymce/tinymce/issues).
+
+> Note: iPads do not use the `mobile` part of the {{site.productname}} init configuration. This is due to a constraint added by Apple to return the environment as a "desktop environment" for iPads. iPads users will receive the other changes to touch functionality, such as context toolbars and context menus.
```

Comment:
'return the environment' kinda makes sense but it's a bit awkward

Reply:
`...constraint added by Apple, setting the environment as...`

Reply:
Actually, this note is no longer true since `mobile` works off the `isTouch` detection which we found a way to work during init right before we released 5.1. Sorry I forgot to update the JIRA @tylerkelly13! :disappointed:

Reply:
One thing to note however, is that `iPads` won't be treated as a `tablet` or `phone` in the browser detection (eg `tinymce.Env.deviceType.isTablet()` will return false)

Reply:
removed